app/inpage/ts/inpage.ts

```diff
@@ -447,14 +472,20 @@ class InterceptorMessageListener {
 			this.WindowEthereumRequest({ method: 'eth_accounts_reply', params: [[], false] })
 		})
 		window.ethereum.on('chainChanged', (chainId: string) => {
-			this.WindowEthereumRequest({ method: 'signer_chainChanged', params: [chainId] })
+			if (/\d/.test(chainId)) { //TODO: this is a hack to get coinbase working that calls this numbers in base 10 instead of in base 16
```

Comment:
could be simplified
```ts
			//TODO: this is a hack to get coinbase working that calls this numbers in base 10 instead of in base 16
			const params = /\d/.test(chainId) ? [`0x${parseInt(chainId).toString(16)}`] : [chainId]
			this.WindowEthereumRequest({ method: 'signer_chainChanged', params })
```

could also link an issue in github to track